addons/web/static/src/js/views/kanban/kanban_renderer.js

```diff
@@ -301,11 +301,15 @@ var KanbanRenderer = BasicRenderer.extend({
      * @private
      * @param {DocumentFragment} fragment
      * @param {integer} nbDivs the number of divs to append
+     * @param {Object} optional values
```

Comment:
- Wrong variable name
- Put the variable between `[]` to indicate it is optional
- "values" is an useless doc
- Add another parameter line: `@param {string} [options.inlineStyle]`